### PR TITLE
Store the correct author for guides

### PR DESCRIPTION
**Pull request by AndrewVos**
If a new guide is created then the author is current_user
If a new draft of an already published guide is created then the new
author becomes current_user.

image

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -24,4 +24,8 @@ def css_class_if_current(path, class_name)
   def name_for_user(user)
     user.name
   end
+
+  def author_options
+    User.pluck(:name, :id)
+  end
```
Comment (thehenster) [inline app/helpers/application_helper.rb]: 👍

File: app/helpers/guide_helper.rb [hunk 2]
```diff
@@ -15,7 +15,7 @@ def state_label(guide)
   end
 
   def latest_editor_name(guide)
-    guide.latest_edition.user.try(:name).to_s
+    guide.latest_edition.author.try(:name).to_s
```
Comment (thehenster) [inline app/helpers/guide_helper.rb]: Does it make sense to rename this method too? `#latest_author_name`?

**Comment by thehenster** [comment]
Only 1 minor suggestion. LGTM

File: app/helpers/guide_helper.rb (hunk 2 above)
Comment (AndrewVos) [inline app/helpers/guide_helper.rb]: https://github.com/alphagov/service-manual-publisher/pull/227/commits/9d2e94dd8f47c96a6442b251537b0804afa5437b

**Pull request merged**